content/blog/aws-cdk-on-pulumi-1.0/index.md

```diff
@@ -0,0 +1,273 @@
+import * as s3 from 'aws-cdk-lib/aws-s3';
+import { AwsSolutionsChecks } from 'cdk-nag';
+
+const app = new pulumicdk.App('app', (scope: pulumicdk.App): pulumicdk.AppOutputs => {
+    const stack = new pulumicdk.Stack('example-stack');
+    Aspects.of(stack).add(new AwsSolutionsChecks({ verbose: true }));
+
+    new s3.Bucket(this, 'bucket');
+});
+```
+
+#### Example Output
+
+```console
+[Error at /test-stack/bucket/Resource] AwsSolutions-S1: The S3 Bucket has server access logs disabled. The bucket should have server access logging enabled to provide detailed records for the requests that are made to the bucket.
+[Error at /test-stack/bucket/Resource] AwsSolutions-S10: The S3 Bucket or bucket policy does not require requests to use SSL. You can use HTTPS (TLS) to help prevent potential attackers from eavesdropping on or manipulating network traffic using person-in-the-middle or similar attacks. You should allow only encrypted connections over HTTPS (TLS) using the aws:SecureTransport condition on Amazon S3 bucket policies.
+```
+
+### CDK assets
+
+CDK's assets make it super easy to bundle code and other files your application needs into an S3 bucket or a Docker container, eliminating extra steps from your build pipeline. For example you can use Docker container asset to deploy an Fargate service from local directory. (CDK will build the container image and push to ECR on `pulumi up`.):
+
+```ts
+import * as pulumicdk from '@pulumi/cdk';
+import * as aws from '@pulumi/aws';
+import * as ec2 from 'aws-cdk-lib/aws-ec2';
+import * as ecs from 'aws-cdk-lib/aws-ecs';
+import * as ecs_patterns from 'aws-cdk-lib/aws-ecs-patterns';
+
+const app = new pulumicdk.App('app', (scope: pulumicdk.App) => {
+    const stack = new pulumicdk.Stack('example-stack');
+
+    const vpc = new ec2.Vpc(stack, 'MyVpc');
+    const cluster = new ecs.Cluster(stack, 'fargate-service-autoscaling', { vpc });
+
+    // Create Fargate Service
+    const fargateService = new ecs_patterns.NetworkLoadBalancedFargateService(this, 'sample-app', {
+        cluster,
+        taskImageOptions: {
+            image: ecs.ContainerImage.fromAsset(path.join(__dirname, './'), {
+                // assetName is required and is used in the name of the ecr repository that is created
+                assetName: 'cdk-fargate-example',
+            }),
+        },
+    });
+});
+```
+
+## Future direction
+
+Currently AWS CDK on Pulumi is supported only for TypeScript users, but we're eager to bring the benefits of the CDK ecosystem to all Pulumi languages in the future. In the meantime, AWS CDK on Pulumi can be used within Component Packages implemented in TypeScript, and exposed to any Pulumi language.
+
+We're also exploring options for importing state from CDK applications that are already deployed with CloudFormation to enable seamless migration for existing CDK users. Please let us know if this is something you would be interested in in testing with us.
```

Comment:
This could use a contact link, e.g. "upvote this issue" or how to reach us with this.